chromadb/db/mixins/sysdb.py

```diff
@@ -222,6 +225,7 @@ def create_collection(
         collection = Collection(
```

Comment:
Why do we hoist this up to a `Collection` again?

Reply:
This is a `CollectionModel` - https://github.com/chroma-core/chroma/issues/2292#issuecomment-2197562289